android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/ViewGenerator.kt

```diff
@@ -146,7 +146,6 @@ fun GenerateView(
             if (isWeighted) Arrangement.spacedBy(properties.spacedBy.dp)
             else properties.arrangement?.position ?: Arrangement.Start
         ) {
-          children.any { it.weight > 0 }
```

Comment:
Why did we remove this?

Reply:
It seemed redundant since its result is not  being used

Reply:
good catch, removing makes sense to me